lib/Runtime/Debug/TTExecutionInfo.h

```diff
@@ -159,6 +162,30 @@ namespace TTD
         void ClearExceptionOnly();
     };
 
+    //A by value class that we use to track the last statement to hit (and abort at) in the inner loop replay
+    class TTInnerLoopLastStatementInfo
+    {
+    private:
+        //The time aware parts of this location
+        int64 m_etime;  //-1 indicates an INVALID location
```

Comment:
use `m_eventTime`, `m_functionTime` and `m_loopTime` here.  And add some comments about what these values mean.